lxqt-config-session/translations/lxqt-config-session_el_GR.ts

```diff
@@ -1,35 +1,35 @@
-<?xml version="1.0" encoding="utf-8"?>
+<?xml version='1.0' encoding='utf-8'?>
 <!DOCTYPE TS>
 <TS version="2.1" language="el_GR">
 <context>
     <name>AutoStartEdit</name>
     <message>
-        <location filename="../autostartedit.ui" line="14"/>
+        <location line="14" filename="../autostartedit.ui"/>
```

Comment:
I will try maybe to compile lxqt with the new translation so it will regenerate the file with this syntax (?) 
Normaly it is nt a problem as in next lupdate they will be changed again. 
This is due the Lokalize program which i use for translations because i have a big translations memory/glossary etc. and it helps me a lot to translate fast.